core/trino-main/src/main/java/io/trino/operator/OperatorContext.java

```diff
@@ -510,9 +511,12 @@ public List<OperatorStats> getNestedOperatorStats()
                 .orElseGet(() -> ImmutableList.of(getOperatorStats()));
     }
 
-    public static Metrics getOperatorMetrics(Metrics operatorMetrics, long inputPositions)
+    public static Metrics getOperatorMetrics(Metrics operatorMetrics, long inputPositions, double cpuTimeSeconds, double wallTimeSeconds)
     {
-        return operatorMetrics.mergeWith(new Metrics(ImmutableMap.of("Input distribution", TDigestHistogram.fromValue(inputPositions))));
+        return operatorMetrics.mergeWith(new Metrics(ImmutableMap.of(
+                "Input distribution", TDigestHistogram.fromValue(inputPositions),
+                "CPU distribution (s)", TDigestHistogram.fromValue(cpuTimeSeconds),
```

Comment:
Maybe `CPU time distribution (s)` ? Just "CPU distribution" could be misinterpreted as vcores count